assets/scripts/fe/src/app/commons/services/payroll/payroll.service.ts

```diff
@@ -13,6 +13,7 @@ import { downloadFileHanlder } from '../../utils/file.utils';
 })
 export class PayrollService {
   public plist : any = [];
+  public mapCheckedPayroll = new Map();
```

Comment:
poor naming convention. explanation is in your other PR.

Do not make 2 variables who does the same thing.